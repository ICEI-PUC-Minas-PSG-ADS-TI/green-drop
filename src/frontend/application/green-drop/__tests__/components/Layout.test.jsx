import React from 'react';
import { render } from '@testing-library/react-native';
import renderer from 'react-test-renderer';
import Layout from '../../layout/index.jsx';

describe('<Layout />', () => {
it('renderiza sem crashar', () => {
render(<Layout />);
});

it('combina com snapshot', () => {
const tree = renderer.create(<Layout />).toJSON();
expect(tree).toMatchSnapshot();
});
});
