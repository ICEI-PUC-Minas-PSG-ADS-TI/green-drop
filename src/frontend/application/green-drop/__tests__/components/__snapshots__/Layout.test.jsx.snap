// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> combina com snapshot 1`] = `null`;
